src/util/map.test.js

```diff
@@ -294,4 +294,24 @@ describe("util/map", () => {
       ]);
     });
   });
+  describe("MapUtil.pushValue", () => {
```

Comment:
Test when the value is an empty array?

Test that array identity is preserved?